projects/chat-lib/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@project-sunbird/chatbot-client",
-  "version": "3.0.1",
+  "version": "3.0.2",
```

Comment:
3.0.2 is already published. Change it to 3.0.3

Reply:
@vinukumar-vs  Package version updated